test/integration/lib/upnpserver_base.js

```diff
@@ -0,0 +1,43 @@
+/* global module */
+"use strict";
+
+const Server = require('node-ssdp').Server;
+
+class upnpServer {
+  constructor(parameters) {
+    this.parameters = parameters;
+    console.log(parameters);
+    this._server = new Server(parameters);
+  }
+
+  createServer() {
+    // Define callbacks
+    console.log("Defining callbacks");
```

Comment:
oh this, it's because upnpserver_base.js is not being used yet, because philips hub is stil Nupnp